cmd/util/util.go

```diff
@@ -210,3 +215,74 @@ minikube config set WantKubectlDownloadMsg false
 			verb, fmt.Sprintf(installInstructions, constants.DefaultKubernetesVersion, goos, runtime.GOARCH))
 	}
 }
+
+type SX map[string]interface{}
+
+func StoreProcessToFile(p os.Process, filename string) error {
+	encoded := ToGOB64(map[string]interface{}{
+		filepath.Base(filename): p,
+	})
+	return ioutil.WriteFile(filename, []byte(encoded), 0644)
+}
+
+func ReadProcessFromFile(filename string) (os.Process, error) {
```

Comment:
At the time of writing this code I had thought that it would not be sufficient for windows to store the PID.  You could be correct though in that all that needs to be stored is the PID, I will test on Windows and add the result here.